app/src/components/navbar/NavbarComponent.jsx

```diff
@@ -77,6 +77,15 @@ function NavbarComponent() {
                 <Link to="/blogs/Durgamandir" className="font-15 text-brown font-regular dropdown-item">
                   Durga Mata Mandir
                 </Link>
+                <Link to="/blogs/VaranasiItinary" className="font-15 text-brown font-regular dropdown-item">
+                  Varanasi Itinary
+                </Link>
+                <Link to="/blogs/VarnasiAyodhyaAllahbad" className="font-15 text-brown font-regular dropdown-item">
+                  Varansi Ayodhya Allahbad
```

Comment:
Name should be same as page name. Page name detailed is suggessted earlier.

Reply:
ok